snap/info_snap_yaml.go

```diff
@@ -53,6 +53,9 @@ type snapYaml struct {
 	Apps             map[string]appYaml     `yaml:"apps,omitempty"`
 	Hooks            map[string]hookYaml    `yaml:"hooks,omitempty"`
 	Layout           map[string]layoutYaml  `yaml:"layout,omitempty"`
+
+	// typeLayouts "layouts" exists to detect the incorrect plural form of "layout".
```

Comment:
Typo in `typ*e*Layouts`- also maybe swap the sentence to get to the point earlier: `typoLayouts is used to detect the use of the incorrect plural form of "layout"`